web/app/model/Attribute.js

```diff
@@ -27,5 +27,11 @@ Ext.define('Traccar.model.Attribute', {
     }, {
         name: 'value',
         type: 'string'
+    }, {
```

Comment:
I guess we can add some fields, but why do we need 3 of them (name, attribute and alias). Don't we need just two (original attribute and alias)?